### ISSUE TITLE
Checksum of OriginSetup.exe changed

### ISSUE THREAD

**Issue by RealTehreal** [bottlesdevs/programs#15]
Hi,

I tried to install Origin using the built-in installer. This is what appeared on the terminal:
```
2021-12-10 15:16:21 ERROR: Downloaded file [OriginSetup.exe] looks corrupted.
2021-12-10 15:16:21 ERROR: Source cksum: [ac0bb3c77786156c22d7ed971737d625] downloaded: [abcf44f57f3f1df87fc87e4e19ba2623]
2021-12-10 15:16:21 INFO: Removing corrupted file [OriginSetup.exe].
2021-12-10 15:16:21 INFO: Setting Key: [dxvk] to [True] for bottle: [origin]…
2021-12-10 15:16:21 INFO: Setting Key: [esync] to [True] for bottle: [origin]…
2021-12-10 15:16:21 INFO: Setting Key: [discrete_gpu] to [True] for bottle: [origin]…
```
Seems like the checksum of the setup file changed.

Greets

**Comment by mirkobrombin** [comment]
Thanks for reporting

**Comment by mirkobrombin** [comment]
Fixed by f26e30ee53d7607e617c6de3c0b247c255c97171
